game_view.cpp

```diff
@@ -260,6 +261,17 @@ void game_view::show() noexcept
     }
 
 
+    if (1 == 2)
```

Comment:
Will this condition ever be reached?

Reply:
No, this is clearly to indivate 'This is a work in progress'. It was what I added at the session days after this Pull Request to show some code how to draw text for Theo. Would the review have been earlier (no need to apologize!) it would not have been.